spopt/tests/data/example_subject_schools.csv

```diff
@@ -0,0 +1,75 @@
+,SE2 PP: Code,SE2 PP: PC,PL: Subject,Count,priority
```

Comment:
I am wondering if it is OK to be using this IOE data here (for privacy, etc.). How much of this example school data is synthetic? In general, I think it is great to have real-world tests. However, the output in `example_subject_student_school_journeys.csv` is rather large to be storing in the repo.

@ljwolf & @gegen07 Any thoughts?

Reply:
It should be all synthetic. Nick also shows the data in a public repository for IOE's case. I can ask him for confirmation later!

Reply:
Maybe we can think about a repo to retrive the datasets that are used in our tests. I think `libpysal.examples` is deprecated, right? We can open a PR to include the datasets to [geodatasets](https://github.com/geopandas/geodatasets) repo which is under geopandas org. What do you think?

Reply:
Hmmm. Yeah, it's listed in [items to deprecate](https://github.com/pysal/libpysal/issues/484). But there's also [continuing activity in that regard](https://github.com/pysal/pysal/issues/1276). Perhaps we can get @sjsrey's input here?

Reply:
@ljwolf What do we think about keeping the testing data as is for this PR and opening a ticket for potential updating in the future? The 3 `.csv` files are a combined ≈187 KB, so def not back-breakers.